docs/book/src/quick-start.md

```diff
@@ -223,6 +223,6 @@ Now, see the [architecture concept diagram][architecture-concept-diagram] for a
 [pre-rbc-gke]: https://cloud.google.com/kubernetes-engine/docs/how-to/role-based-access-control#iam-rolebinding-bootstrap
 [cronjob-tutorial]: https://book.kubebuilder.io/cronjob-tutorial/cronjob-tutorial.html
 [GOPATH-golang-docs]: https://golang.org/doc/code.html#GOPATH
-[go-module-blogpost]:https://blog.golang.org/using-go-modules
+[go-module-blogpost]: https://blog.golang.org/using-go-modules
```

Comment:
I found that `go-module-blogpost` also needs to be modified to `go-modules-blogpost`.
https://github.com/kubernetes-sigs/kubebuilder/blob/b6ecf5ef413852d8ca2a7b0281c54e316dbb5b29/docs/book/src/quick-start.md#L68

Reply:
In the plural is better. What do you think?

Reply:
Yes:smiley: